charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -43,6 +43,6 @@ dependencies:
   repository: https://prometheus-community.github.io/helm-charts
   condition: nodeExporter.enabled
 - name: grafana
-  version: "6.19.*"
+  version: "6.20.*"
```

Comment:
Can you run `helm dependency update` as well?

Reply:
done